### PR TITLE
#5351 [Manager] Should not allow shared topic from adding into Paper …

### PR DESCRIPTION
**Pull request by konekoya**
…if the name is already taken by other elements

### Checklist

- [x] [Review Contribution Guidelines](https://oharastream.github.io/en/docs/master/contributing/)
- [x] Add at least one reviewer (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add an assignee (of course, you are the assignee by default)
- [x] Do not set milestone, projects and labels (PR labels are added automatically by Jenkins)
- [x] Link to an issue
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (If the failed tests are unrelated to this PR, add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/cypress/it/paper/toolbox.test.ts [hunk 1]
```diff
@@ -285,6 +285,54 @@ describe('ToolBox', () => {
...
+      cy.createSharedTopic(name);
+
+      // Add a sink into Paper with the same name
+      cy.addElement(name, KIND.sink, SINKS.shabondi);
+
+      // Then, add the topic
+      cy.findByText(Cypress._.capitalize(KIND.topic)).should('exist').click();
+
+      // Add topic
+      cy.findByTestId('toolbox-draggable')
```
Comment (konekoya) [inline ohara-manager/client/cypress/it/paper/toolbox.test.ts:317]: `cy.addElement` command does a check to see if the added elemenet exists or not. So we cannot use the command here as the Element won't be added

File: ohara-manager/client/cypress/it/paper/toolbox.test.ts [hunk 2]
```diff
@@ -285,6 +285,54 @@ describe('ToolBox', () => {
...
+    it('should not add an element if the element is over Toolbox', () => {
+      // Open topic panel
+      cy.findByText(Cypress._.capitalize(KIND.topic)).should('exist').click();
+
+      // Add a pipeline-only topic
+      cy.findByTestId('toolbox-draggable')
+        .find(`g[data-type="${CELL_TYPES.ELEMENT}"]:visible`)
+        // the only "draggable" cell is pipeline-only topic
+        .first()
+        .dragAndDrop(30, 450);
```
Comment (saivirtue) [inline ohara-manager/client/cypress/it/paper/toolbox.test.ts:298]: first user 🙌

Comment (konekoya) [reply ohara-manager/client/cypress/it/paper/toolbox.test.ts:298]: It's quite handy XD

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**